app/src/main/java/com/infomaniak/mail/data/models/Folder.kt

```diff
@@ -86,6 +88,11 @@ class Folder : RealmObject {
     val role: FolderRole?
         get() = enumValueOfOrNull<FolderRole>(_role)
 
+    // `> 0` should display the number.
+    // `-1` should display a pastille.
+    // `0` should display nothing.
+    val unreadCountToDisplay: Int get() = if (unreadCountLocal > 0) unreadCountLocal else if (unreadCountRemote > 0) -1 else 0
```

Comment:
I feel like we could use a data class for the badge to have the text and the state (hidden, pastille, displayed or smt like that) in one object to avoid using magic numbers and beautify the code.

Could probably be overkill though